fileio/fileio_test.go

```diff
@@ -55,13 +57,28 @@ func TestEqual(t *testing.T) {
 }
 
 func TestMustCreateEmptyFilenameError(t *testing.T) {
-	if os.Getenv("TEST_FATAL") != "1" {
-		return
-	}
+	defer func() {
+		if r := recover(); r == nil {
+			t.Errorf("Error: Unknown file type should throw error...\n")
+		}
+	}()
+
 	// Will cause log.Fatal in this test context
 	MustCreate("")
-}
 
+	var buf bytes.Buffer
```

Comment:
didn't like my previous test for catching empty string error. this test captures log error if inputing a blank string